docs/docsite/rst/network/user_guide/faq.rst

```diff
@@ -0,0 +1,34 @@
+.. _network_faq:
+
+*******************
+Ansible Network FAQ
+*******************
+
+How can I improve performance for network playbooks?
+====================================================
+
+* Set ``strategy: free`` at the playbook level if you are running on multiple hosts
+
+The ``strategy`` plugin controls the ordering of tasks on hosts. When ``strategy=linear``, Ansible will wait until the current task has run on all hosts before starting the next host. When ``strategy=free``, Ansible will execute the next task on each host as soon as that host completes the current task, so overall execution is not waiting for the slowest host. 
+
+* Execute ``show running`` only if you absolutely must
+
+The ``show running`` command is the most resource-intensive command to execute on a network device, because of the way queries are handled by the network OS. Using the command in your Ansible playbook will slow performance significantly, especially on large devices; repeating it will multiply the performance hit. If you have a playbook that checks the running config, then executes changes, then checks the running config again, you should expect that playbook to be very slow.
+
+* Use ``ProxyCommand`` only if you absolutely must
+
+Network modules support the use of a jump host with the ``ProxyCommand`` parameter. However, when you use a jump host, Ansible must open a new SSH connection for every task, even if you are using a persistent connection type (``network_cli`` or ``netconf``). To maximize the performance benefits of the persistent connection types introduced in version 2.5, avoid using jump hosts whenever possible.
```

Comment:
Link to proxy details

Reply:
I'm not sure what page you mean by "proxy details" - are you thinking of here -http://docs.ansible.com/ansible/devel/network/user_guide/network_debug_troubleshooting.html#proxy-issues ? Or here - http://docs.ansible.com/ansible/devel/user_guide/playbooks_environment.html?highlight=proxy ?